git_machete/utils.py

```diff
@@ -183,24 +183,24 @@ def mark_current_directory_as_possibly_non_existent() -> None:
 def chdir_upwards_until_current_directory_exists() -> None:
     global current_directory_confirmed_to_exist
     if not current_directory_confirmed_to_exist:
-        current_directory: Optional[str] = current_directory_or_none()
+        current_directory: Optional[str] = get_current_directory_or_none()
         if not current_directory:
             while not current_directory:
                 # Note: 'os.chdir' only affects the current process and its subprocesses;
                 # it doesn't propagate to the parent process (which is typically a shell).
                 os.chdir(os.path.pardir)
-                current_directory = current_directory_or_none()
+                current_directory = get_current_directory_or_none()
             debug("chdir_upwards_until_current_directory_exists()",
                   f"current directory did not exist, chdired up into {current_directory}")
         current_directory_confirmed_to_exist = True
 
 
-def popen_cmd(cmd: str, *args: str, **kwargs: Any) -> Tuple[int, str, str]:
+def get_popen_cmd(cmd: str, *args: str, **kwargs: Any) -> Tuple[int, str, str]:
```

Comment:
I'd leave `popen_cmd` to be honest. This method still might have side effects (caused by the command getting executed), it isn't a side-effect-free one like, say, `get_pretty_choices` above